borg-mixin/prometheus-alerts.yaml

```diff
@@ -0,0 +1,11 @@
+"groups":
```

Comment:
Maybe add a pre-generated dashboard as well. Or I can do it in a diff PR.

Reply:
Done